### DISCUSSION TITLE
[Request] have a toggle for auto pinch to zoom

### DISCUSSION THREAD

**Discussion by AdiA105** [qnblackcat/uYouPlus#563] [Ideas]
When you open the YouTube app and play a video you have to pinch to zoom. When you close the mini player and play another video you have to pinch to zoom again. It would be nice if the app remembered that i like to watch videos zoomed, or add a toggle.

**Comment by ErSpieltHD** [comment]
@AdiA105 Go to YouTube Settings > General > _Zoom to screen size (i don't know how it's exactly called in English)_

**Reply by AdiA105** [reply]
it helps when I close the miniplayer and play another video but not if I relaunch the app. So it half works but thank you so much!

**Reply by ErSpieltHD** [reply]
It doesn't work for all videos. Not even in the stock (original) YT app. (At least for me)

**Reply by ErSpieltHD** [reply]
Btw, this feature is from YT directly and not from uYouPlus.
